.changeset/gentle-spoons-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-type-no-unknown` false positives for custom identifiers passed to `::highlight()` and `::view-transition-*()`
```

Comment:
```suggestion
Fixed: `selector-type-no-unknown` false positives for `::highlight()` and `::view-transition-*()`
```

Same for the PR and issue title.

-----

The `custom-ident` aspect is actually irrelevant and inaccurate :)
These pseudo's do not take selectors as their argument.
Although those arguments do sometimes look like selectors.

This is also valid for example: `::view-transition-group(*)`, where `*` is also not a selector.

So we ignore them because their arguments aren't selectors, not because their arguments might be custom-idents.

Simplifying the changelog entry makes it more correct :)